surveys/2024-annual-survey/questions.md

```diff
@@ -369,6 +369,34 @@ Type: free text
 >
 > Allow the cohort to mention specific language features they might be eagerly waiting for, see https://github.com/rust-lang/surveys/pull/234/files#r1347633041
 
+### Which features stabilized in the past 12 months do you use the most?
+
+The text in the parentheses shows in which Rust version was the feature stabilized.
+
+Type: matrix (optional)
+
+Features:
+
+- `async fn` in traits (1.75)
+- Return position `impl Trait` in traits (1.75)
+- C-string literals (1.77)
+- Recursion in `async fn` (1.77)
+- `offset_of!` macro (1.77)
+- `#[diagnostic]` attribute (1.78)
+- Inline `const` expressions (1.79)
+- Bounds in associated type position (1.79)
+- `LazyCell` and `LazyLock` (1.80)
+- Exclusive ranges in patterns (1.80)
+- `#[expect(lint)]` and lint reasons (1.81)
```

Comment:
I have added links to the relevant blog post announcements, hopefully people can quickly understand what each option is about. Sometimes I use things without even knowing what they are. Like my heart and lungs.

Can Survey Hero parse these links?

```suggestion
- `async fn` in traits ([1.75](https://blog.rust-lang.org/2023/12/28/Rust-1.75.0.html))
- Return position `impl Trait` in traits ([1.75](https://blog.rust-lang.org/2023/12/28/Rust-1.75.0.html))
- C-string literals ([1.77](https://blog.rust-lang.org/2024/03/21/Rust-1.77.0.html))
- Recursion in `async fn` ([1.77](https://blog.rust-lang.org/2024/03/21/Rust-1.77.0.html))
- `offset_of!` macro ([1.77](https://blog.rust-lang.org/2024/03/21/Rust-1.77.0.html))
- `#[diagnostic]` attribute ([1.78](https://blog.rust-lang.org/2024/05/02/Rust-1.78.0.html))
- Inline `const` expressions ([1.79](https://blog.rust-lang.org/2024/06/13/Rust-1.79.0.html))
- Bounds in associated type position ([1.79](https://blog.rust-lang.org/2024/06/13/Rust-1.79.0.html))
- `LazyCell` and `LazyLock` ([1.80](https://blog.rust-lang.org/2024/07/25/Rust-1.80.0.html))
- Exclusive ranges in patterns ([1.80](https://blog.rust-lang.org/2024/07/25/Rust-1.80.0.html))
- `#[expect(lint)]` and lint reasons ([1.81](https://blog.rust-lang.org/2024/09/05/Rust-1.81.0.html))
```

Reply:
Hmm, I thought that it's not possible, since last year we also wanted to have links for the "features that should be stabilized" question, but it was not possible for some reason. But now it looks like it can be done in SurveyHero. So yeah, we can do this.

Reply:
Added the corresponding links.